tests/ci/script.sh

```diff
@@ -30,22 +30,19 @@ if [ "${ACMS_JOB}" == "backstop_tests" ] || [ "${ACMS_JOB}" == "upgrade_modules"
   orca fixture:run-server &
 
   # Runs Backstop.js
-  npm run backstop-starter
-
-  # Generate backstop test images.
-  # npm run backstop-starter && error=false || error=true
-  # if [ "${error}" = "true" ]; then
-  #   ./node_modules/.bin/backstop reference --config=tests/backstop/backstop-settings.js
-  #   ./node_modules/.bin/backstop approve --config=tests/backstop/backstop-settings.js
-  #   aws s3 cp --recursive "./tests/backstop/bitmaps_reference/" "${AWS_S3_BUCKET_PATH}/backstop/reference/${GITHUB_RUN_ID}"
-  # fi
-
-  # Store failed backstop test images.
-  # npm run backstop-starter && error=false || error=true
-  # if [ "${error}" = "true" ]; then
-  #    tar -cvzf test_reports.tar.gz ./tests/backstop/bitmaps_test/;
-  #    aws s3 cp --recursive "./tests/backstop/bitmaps_test/" "${AWS_S3_BUCKET_PATH}/backstop/logs/${GITHUB_RUN_ID}"
-  # fi
+  npm run backstop-starter && error=false || error=true
+  if [ "${error}" = "true" ]; then
+    if [ "${COPY_BACKSTOP_IMAGES}" == "true" ]; then
+      tar -cvzf test_reports.tar.gz ./tests/backstop/bitmaps_test/;
+      aws s3 cp --recursive "./tests/backstop/bitmaps_test/" "${AWS_S3_BUCKET_PATH}/backstop/logs/${GITHUB_RUN_ID}"
```

Comment:
Any reason we are sending backstop test failure images to AWS all time.

Reply:
We are not sending all the time. Default value for `COPY_BACKSTOP_IMAGES ` is false: https://github.com/acquia/acquia_cms/pull/1802/files#diff-4eb666b493503cabe7a4f72501ebd718bc71bc8560928ffc0b9b9d031f15c8e1R230
Developer can just set this value as true if they need to copy to AWS and see.

Reply:
awesome 🙌